docs/5g-on-helium/5g-poc.mdx

```diff
@@ -165,16 +166,28 @@ Network switches that are advertised as "100/10" switches are often "90/9" in re
 
 :::
 
-### Genesis Phase 4: Reward Frequency Increase
+### Genesis Phase 4: Heartbeats and Reliability of Service
 
-> - Reward Periods shortened to 2 Epochs (60 blocks, or approximately 1 hour).
-> - Rewards transactions are triggered with every other Consensus Election.
+> - Reward Period is approximately 1440 blocks and closely aligns with 00:00 UTC to 23:59 UTC.
+> - Reward submission occurs approx 15 minutes after the Reward Period ends, around 00:00 UTC.
+> - 5G Hotspots perform periodic speed tests which will be considered in reward calculations.
+> - 5G Hotspots must generate 1 "heartbeat" in a 1 hour period for at least 12 periods to ensure it
+>   is providing reliable coverage uptime.
 
-### Genesis Phase 5: Reward Period Decoupling
+In addition to speed test requirements, a 5G Hotspot is required to generate at least 1 "heartbeat"
+per hour in each of at least 12 unique hours in the Reward Period.
 
-> - Decouple Reward Period From Reward Submission Time.
-> - Increase tolerance for delay of heartbeat and speed test data by allowing rewards to be sent for
->   the receipts that came in late.
+At this Milestone, to qualify for the Genesis Rewards in a Reward Period, a CBRS Small Cell Radio
```

Comment:
It feels that we repeat here what was already mentioned above, maybe we can replace some of the duplication with the examples.